buildconfig/pygame-stubs/mixer.pyi

```diff
@@ -12,19 +13,19 @@ from . import music as music
 import numpy
 
 def init(
-    frequency: Optional[int] = 44100,
-    size: Optional[int] = -16,
-    channels: Optional[int] = 2,
-    buffer: Optional[int] = 512,
-    devicename: Optional[Union[str, None]] = None,
-    allowedchanges: Optional[int] = 5,
+    frequency: int = 44100,
+    size: int = -16,
+    channels: int = 2,
+    buffer: int = 512,
+    devicename: Optional[str] = None,
+    allowedchanges: int = 5,
 ) -> None: ...
 def pre_init(
-    frequency: Optional[int] = 44100,
-    size: Optional[int] = -16,
-    channels: Optional[int] = 2,
-    buffer: Optional[int] = 512,
-    devicename: Optional[Union[str, None]] = None,
+    frequency: int = 44100,
+    size: int = -16,
+    channels: int = 2,
+    buffer: int = 512,
+    devicename: Optional[str] = None,
```

Comment:
pre_init also needs the allowedchanges argument. The docs didn't have that correct until recently, but the docs source has been fixed.

Reply:
Ooh nice to know, will fix the typehint in this PR